Source/GmmLib/inc/External/Common/GmmCachePolicy.h

```diff
@@ -52,16 +52,20 @@ typedef struct GMM_CACHE_POLICY_ELEMENT_REC
             uint32_t                   SSO         : 2; // Self Snoop Override  control and value
             uint32_t                   HDCL1       : 1; // HDC L1 caching enable/disable
             uint32_t                   L3Eviction  : 2; // Specify L3-eviction type (NA, ReadOnly, Standard, Special)
+            uint32_t                   SegOv       : 3; // Override seg-pref (none, local-only, sys-only, etc)
```

Comment:
This breaks ABI compatibility while you can easily preserve it. The rule of thumb is: if you borrow values from reserved bits, add them to the end to avoid shifting of the existing bits positions.

Reply:
will look into this, need to make sure if 'Initialized' field was placed at the end to follow specific design guideline, will get back on this.

Reply:
will move it towards the end.  we do not work directly with bit positions, and work with variables so will not cause any harm to shift it towards end.

Reply:
I think you forgot to remove this one. you have duplicate below - this won't build.